app/assets/stylesheets/nfg_ui/network_for_good/core/nfg_theme/_alert.scss

```diff
@@ -6,13 +6,28 @@
     &:hover, &:active { text-decoration: none; }
   }
 }
-.alert-success a:not(.btn) { color: $success; }
-.alert-info a:not(.btn) { color: $info; }
-.alert-warning a:not(.btn) { color: $warning; }
+
+@each $color, $value in $theme-colors {
+  .alert-#{$color} {
+    background: scale-color($value, $lightness: 80%);
+  }
+}
+.alert-success a:not(.btn) { color: $success !important; }
+.alert-info a:not(.btn) { color: $info !important; }
+.alert-warning a:not(.btn) { color: $warning !important; }
 .alert-danger, .alert-error {
-  a:not(.btn) { color: $danger; }
+  a:not(.btn) { color: $danger !important; }
 }
 .alert-gray, .alert-notice,
 .alert-light, .alert-dark {
   a:not(.btn) { color: inherit; }
 }
+
+// Absolute Positioning Alerts
+.alert[data-position-absolute='true'] {
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
+  z-index: $zindex-modal + 1;
+  box-shadow: $box-shadow-sm;
+}
```

Comment:
@kylejmorin / @farhajawed another one we may want to consider adding a pattern for -- displaying absolute positioning alerts. These aren't widely used, i'll leave this up to @kylejmorin to determine if we're going to keep this as a pattern moving forward or not :)

Reply:
Going forward, i think these absolute positioned alerts can be moved to toasts.